ipa-core/src/test_fixture/world.rs

```diff
@@ -99,43 +156,29 @@ impl Default for TestWorld {
     }
 }
 
-impl TestWorld {
+impl<const SHARDS: usize> TestWorld<Sharded<SHARDS>> {
+    #[must_use]
+    pub fn with_shards<B: Borrow<TestWorldConfig>>(config: B) -> Self {
+        Self::with_config(config.borrow())
+    }
+
+    fn shards(&self) -> [&ShardWorld<Shard>; SHARDS] {
+        self.shards
+            .iter()
+            .collect::<Vec<_>>()
+            .try_into()
+            .unwrap_or_else(|_| unreachable!())
```

Comment:
I believe that just `unwrap()` will achieve nearly the same thing.  Is this due to a lack of a Debug impl?

Reply:
yep, exactly

Reply:
I've been using `.ok().unwrap()` in this situation. (Not worth an edit just to change that, more of a side note.)

Reply:
`ok().unwrap()` is strictly better, worth changing it